README.md

```diff
@@ -88,6 +88,8 @@ Applications with support of IPTV streams.
 #### Android
 
 - [Kodi](https://play.google.com/store/apps/details?id=org.xbmc.kodi) - Free cross-platform media player with library support.
+- [IPTVmine](https://github.com/samyak2403/IPTVmine/raw/master/app/release/app-release.apk) - The Indian IPTV App is a comprehensive platform that allows users to stream over 500 Indian TV channels directly from their devices. .
```

Comment:
> - new links are added to the end of the section

https://github.com/iptv-org/awesome-iptv/blob/master/.github/CONTRIBUTING.md#how-do-i-add-a-new-link-to-the-list